modoboa/core/api/v2/serializers.py

```diff
@@ -126,7 +127,7 @@ class CoreGlobalParametersSerializer(serializers.Serializer):
     log_maximum_age = serializers.IntegerField(default=365)
     items_per_page = serializers.IntegerField(default=30)
     default_top_redirection = serializers.ChoiceField(
-        default="user", choices=[""])
+        default="user", choices=app_settings.enabled_applications(), required=False)
```

Comment:
Calling this here is not a good idea because it will be executed at import time and we can't guarantee that all modules will be loaded. You should move this in the serializer constructor.

Reply:
Done, thanks